runners/apex/src/main/java/org/apache/beam/runners/apex/translators/utils/ApexStreamTuple.java

```diff
@@ -109,6 +109,33 @@ public long getTimestamp() {
     public void setTimestamp(long timestamp) {
       this.timestamp = timestamp;
     }
+
+    @Override
+    public int hashCode() {
```

Comment:
I favor `Objects.hash` and `Objects.equals` to build these things. But for a simple POJO like this, actually consider using `@AutoValue` if you can. (is the setter crucial?)